### ISSUE TITLE
A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader:Missing grouping for "step 1 Select the deployment option screen" three radio buttons.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7667]
**GitHubTags:** #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG3.3.2;#DesktopApp#Win32;#Keyboard;
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
**Repro Steps:**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster”.
5. "Select the deployment option" screen gets open.
6. Start NVDA and navigate with the help of up and down arrow and Observe the announcement.

**Actual:**
Missing grouping for the for the  for "step 1 Select the deployment option screen"
**Expected:**
Groping should be marked for the three radio buttons available on the screen.
**User Impact:**
User will not able to directly relate with the three radio button which is present on the screen. 

Attachment for Reference:

[7667_A11y_AzureDataStudio_Selectthedeploymentoption_Zoom.pptx](https://github.com/microsoft/azuredatastudio/files/3717056/7667_A11y_AzureDataStudio_Selectthedeploymentoption_Zoom.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
this is fixed now, nvda reads 2 of 3 and then the name of the option.
